packages/cardpay-sdk/sdk/utils/signing-utils.ts

```diff
@@ -135,6 +135,55 @@ export async function signSafeTxAsBytes(
   return [await signTypedData(web3, owner, typedData)];
 }
 
+export async function fullSignatureTxAsBytes(
```

Comment:
could we rename this function. it's not clear to me as a caller of this util when I'd want to use `fullSignatureTxAsBytes` vs `signSafeTxAsBytes`. let's change the name of this function to indicate why it's different and when I'd use one vs the other